src/hyperfine/types.rs

```diff
@@ -46,7 +46,7 @@ pub struct Command<'a> {
     expression: &'a str,
 
     /// A possible parameter value.
-    parameter: Option<(&'a str, NumericType)>,
+    parameter: Option<(&'a str, String)>,
```

Comment:
It might even be possible to use a `&'a str` reference here instead of a `String`.